VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/participant-hearings/hearing-list-table.component.html

```diff
@@ -28,18 +28,18 @@
           class="govuk-button"
           data-module="govuk-button"
           type="button"
-          *ngIf="canStartHearing(conference)"
+          *ngIf="!canStartHearing(conference)"
           (click)="signIntoConference(conference)"
-          aria-label="Sign into hearing"
+          attr.aria-label="{{ 'hearing-list-table.button.sign-into-hearing' | translate }}"
         >
-          Sign into hearing
+        {{ 'hearing-list-table.button.sign-into-hearing' | translate }}
         </button>
-        <span *ngIf="!canStartHearing(conference)">
+        <span *ngIf="canStartHearing(conference)">
```

Comment:
is this supposed to have been inverted? (removed leading '!')

Reply:
good spot! :) nope was for testing